File: mobile-client/ios/Contract/Taxation/Detail/TaxDetailPayload.swift
import Foundation

// Marketing America Corp. Oleksandr Tishchenko
struct TaxDetailPayload {
    let jurisdictionCode: String
    let taxableAmountLabel: String
    let subtotalAmountLabel: String
    let taxAmountLabel: String
    let totalAmountLabel: String
    let rateLabel: String
    let breakdownLines: [String]
}

File: mobile-client/android/Contract/Taxation/Detail/TaxDetailPayload.kt
package app.mobiling.client.contract.taxation.detail

/**
 * Marketing America Corp. Oleksandr Tishchenko
 */
data class TaxDetailPayload(
    val jurisdictionCode: String,
    val taxableAmountLabel: String,
    val subtotalAmountLabel: String,
    val taxAmountLabel: String,
    val totalAmountLabel: String,
    val rateLabel: String,
    val breakdownLines: List<String>,
)
